src/types/types.ts

```diff
@@ -162,13 +163,13 @@ export interface ETRNamedTrigger {
   triggerName: string;
 }
 
-export interface ETRLaunchingOnSerialNumber { 
+export interface ETRLaunchingOnSerialNumber {
   machineSerialNumber: string;
 }
 
 export interface ETRMoveMouseToCorner {
-  delayBeforeTriggering: number,
-  allowDragging: 0 | 1,
+  delayBeforeTriggering: number;
```

Comment:
wow, I don't know how I missed it :) thanks

Reply:
we need a tslint config otherwise you get everytime a different formatting by other contributors.
Can i make a setup as a suggestion?

Reply:
Yes, go ahead! We can talk through the rules later, but I'd like to ask you to set `quotemarks` rule to use `single` (alongside   `avoid-escape` and `avoid-template`)

thanks for your time

Reply:
normally i have this too and i was shocked after i created the PR 💯 
But i willdefinitelyy add single quotes